### DISCUSSION TITLE
Does anyone know how to make a script in tampermonkey that changes the keyword "experiences" to "games" again?

### DISCUSSION THREAD

**Discussion by dokiLBY** [anthony1x6000/ROBLOX2016stylus#34] [Q&A]
I'm not really a good coder and i don't know what am i doing.  So i was hoping that someone here might know how to pull this off.

**Comment by anthony1x6000** [comment]
Here's [this](https://github.com/anthony1x6000/ROBLOX2016stylus/blob/main/legacywords.user.js)

It looks through the entire document and replaces words. It works okay but it might slow down the site or break some other things.

**Reply by dokiLBY** [reply]
Oh alright
